README.md

```diff
@@ -67,15 +67,12 @@ Then type path and name of new configuration file.
 
 To create new stack you have to type:
 
-``~ $ perun --mode=create-stack
-             --template=<PATH TO YOUR TEMPLATE>
-             --stack=<NAME OF YOUR STACK>
+``~ $ perun create-stack <PATH TO YOUR TEMPLATE> <NAME OF YOUR STACK>
```

Comment:
Actually, I liked that idea - please restore  parameters as `--stack-name` and `--template-file`.

Reply:
Do You mean the order or actually leaving the named parameters? If the latter, I believe it could be a little confusing since those are obligatory.

Reply:
I know they are, still, you don't know the order and what to put there without named flag or looking up help. I'd vote for restoring those.

Reply:
Ok, I'll change it then, but for now, the modes will be inconsistent then - similar approach has been taken in convert mode etc.

Reply:
You're right, let's leave it as is and I will create new ticket for that.